api/resources/views/filled-form/edit.blade.php

```diff
@@ -0,0 +1,25 @@
+@extends('layouts.app')
+
+@section('content')
+
+    <div class="container mt-4">
+        @include('errors')
+        <form action="{{ route('filled-forms.store') }}" method="post">
+            @csrf
+            @foreach($fields as $field => $status)
+                <div class="form-group">
+                    @if($statuses->get($status) !== 'unused' )
+                        <label style="text-transform: capitalize" for="{{$field}}">{{ implode(" ", explode('_', $field)) }}</label>
+                        <input type="text" class="form-control form-control" id="{{$field}}" name="{{$field}}"
```

Comment:
Would there be a way to validate these fields? Right now I can enter REEE as a phone number, which is not really good. Same with when filling a form.

Reply:
It is jus strings which we should realy care what is inputed, because for egzample we would need regular exp for numbers, lets just leave like rel life paper version of form, you can write whatever you like in it

Reply:
So if I insert JOE MAMA that's a legit response? Okay.